activator

```diff
@@ -1,334 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Nice that you removed the activator related files 👍

Reply:
Yep, we didn't need them anymore since nobody is supposed to run this app.